akka-http-caching/src/main/scala/akka/http/caching/scaladsl/CachingSettings.scala

```diff
@@ -63,14 +63,14 @@ private[http] final case class LfuCacheSettingsImpl(
 }
 
 object CachingSettings extends SettingsCompanion[CachingSettings]("akka.http.caching") {
-  def fromSubConfig(root: Config, c: Config) = {
+  def fromSubConfig(root: Config, c: Config): CachingSettingsImpl = {
```

Comment:
looks like leak of internal type? Should be `: CachingSettings`?

Reply:
(Notice that we are not in the `Impl` object)

Reply:
Thanks, changed.